volume/local/local.go

```diff
@@ -367,6 +367,13 @@ func validateOpts(opts map[string]string) error {
 			return validationError{fmt.Errorf("invalid option key: %q", opt)}
 		}
 	}
+	if len(opts) > 0 {
+		for mKey := range mandatoryKey {
+			if _, ok := opts[mKey]; !ok {
+				return validationError{fmt.Errorf("option %s must be specified", mKey)}
```

Comment:
Can we use `errors.Errorf`? This passes the error with the stack trace for debugging purposes.

Maybe we can re-word this, something like "missing required option %s".